defaults/main.yaml

```diff
@@ -1,9 +1,45 @@
 ---
+system:
```

Comment:
We can use the mariadb namespace but have to introduce those system variables, because we cannot presume that other users of this role use the system role as well.

Reply:
fixed in 37d776de2649152dfadf65ada9ce1845d1176c68

Reply:
Fixed in 37d776de2649152dfadf65ada9ce1845d1176c68

Reply:
It should be possible to use conditional dependencies here (in `meta/main.yaml`).

```yaml
dependencies:
  - role: system
    when: ansible_system == 'Linux'
```

But maybe it's really a better idea to keep it in this role (less dependencies means easier to install and no version conflicts). I just don't want the `mariadb` role to write to a different namespace because this can cause subtle conflicts that are really hard to spot and debug. But if it's in the `mariadb` namespace it should work fine, let's do it that way.